[doc] Align agents logs documentation (#1394)
elastic/apm-agent-ruby: docs/logs.asciidoc

please view this documentation at https://www.elastic.co/guide/en/apm/agent/ruby[elastic.co]
endif::[]

[[logs]]
== Logs

Elastic Ruby APM Agent provides the following log features:

> - <<log-correlation-ids>> : Automatically inject correlation IDs that allow navigation between logs, traces and services.

Reviewer: ```suggestion
- <<log-correlation-ids>>: Automatically inject correlation IDs that allow navigation between logs, traces and services.
```

> - <<log-reformatting>> : Automatically reformat plaintext logs in {ecs-logging-ref}/intro.html[ECS logging] format.

Reviewer: ```suggestion
- <<log-reformatting>>: Automatically reformat plaintext logs in {ecs-logging-ref}/intro.html[ECS logging] format.
```

Those features are part of {observability-guide}/application-logs.html[Application log ingestion strategies].

> The {ecs-logging-ruby-ref}/intro.html[`ecs-logging-ruby`] library can also be used to use the {ecs-logging-ref}/intro.html[ECS logging] format without an APM agent.

Reviewer: ```suggestion
The {ecs-logging-ruby-ref}/intro.html[`ecs-logging-ruby`] library can also be used to format logs in the {ecs-logging-ref}/intro.html[ECS logging] format without an APM agent.
```
